### ISSUE TITLE
whiteboard not detected with OpenImages weights/cfg etc.

### ISSUE THREAD

**Issue by rxjx** [AlexeyAB/darknet#3262]
I'm using cfg/yolov3-openimages.cfg edited to point to data/openimages.names with https://pjreddie.com/media/files/yolov3-openimages.weights to detect whiteboards in images. However, I don't see a single detection hitting. Is this something similar to https://github.com/AlexeyAB/darknet/issues/2575#issuecomment-471784534?

**Comment by AlexeyAB** [comment]
Do you want to detect only `whiteboard` ? https://github.com/AlexeyAB/darknet/blob/master/data/openimages.names#L158

Then just add `dont_show` before each line in the `openimages.names` except `Whiteboard`

**Comment by rxjx** [comment]
Thanks, but that's not the issue. I did find the dont_show option in "Issues" and used it. However, I just don't see any detection at all on images that clearly have whiteboards.

**Comment by AlexeyAB** [comment]
But can you detect `whiteboards` on the same images if you use original `openimages.names` without `dont_show`?

If you can't detect, the yes the issue - bad trained model for whiteboards, the same as here: https://github.com/AlexeyAB/darknet/issues/2575#issuecomment-471784534

**Comment by rxjx** [comment]
Nope, can't detect either way. So probably a badly trained model. Thanks.
